osbuild/libdir.py

```diff
@@ -0,0 +1,37 @@
+import os
+from typing import List
+
+
+class Libdir:
+    """Libdir abstracts the osbuild --libdir handling"""
+
+    def __init__(self, libdir: str) -> None:
+        self._libdir = ":".join(os.path.abspath(p)
+                                for p in os.fspath(libdir).split(":"))
```

Comment:
Is it necessary to call `os.fspath`? `libdir` can't be anything but a `str` already?

Reply:
IIRC some things passed pathlib.Path and your mypy coverage is spotty (at best) - but I removed it now, let's see what breaks (hopefully nothing)